<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDlg</class>
 <widget class="QDialog" name="RegisterDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>645</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>645</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>645</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#RegisterDlg
{
background-color:#f1f3f6;
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>50</y>
     <width>511</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget
{
background-color:#ffffff;
border-radius:50; 
}</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton
{
color:#a7b0ca;
  background-color: rgba(0, 0, 0, 0);
  border: none; 
}

#pushButton:hover{
color:#dd3a5f;
}</string>
    </property>
    <property name="text">
     <string>&lt; 返回</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label
{
color:#1a263c;
}</string>
    </property>
    <property name="text">
     <string>教职工注册</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit
{
/*border:none;*/
border: 1px solid #e5e7e9;
border-left:none;
border-top-right-radius:25;
border-bottom-right-radius:25;
 padding: 8px; 
}

#lineEdit:hover
{
border: 1px solid #f7115b;
border-left:none;
border-top-right-radius:25;
border-bottom-right-radius:25;
 padding: 8px; 
}</string>
    </property>
    <property name="placeholderText">
     <string>请输入职工号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_2
{
/*border:none;*/
border: 1px solid #e5e7e9;
border-left:none;
border-top-right-radius:25;
border-bottom-right-radius:25;
 padding: 8px; 
}

#lineEdit_2:hover
{
border: 1px solid #f7115b;
border-left:none;
border-top-right-radius:25;
border-bottom-right-radius:25;
 padding: 8px; 
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请输入密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>135</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_2
{
color:red;
}</string>
    </property>
    <property name="text">
     <string>错误：职工号输入格式错误</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_3
{
color:#95a5c0;
}</string>
    </property>
    <property name="text">
     <string>职工号只能由数字组成，且不超过16位</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>244</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_4
{
color:red;
}</string>
    </property>
    <property name="text">
     <string>错误：密码输入格式错误</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>441</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_5
{
color:#95a5c0;
}</string>
    </property>
    <property name="text">
     <string>密码要求 6-11位，只能由英文与数字组成且必须包含英文与字母</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_3
{
/*border:none;*/
border: 1px solid #e5e7e9;
border-left:none;
border-top-right-radius:25;
border-bottom-right-radius:25;
 padding: 8px; 
}

#lineEdit_3:hover
{
border: 1px solid #f7115b;
border-left:none;
border-top-right-radius:25;
border-bottom-right-radius:25;
 padding: 8px; 
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>确认密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>430</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_6
{
color:red;
}</string>
    </property>
    <property name="text">
     <string>输入与密码不一致，请重新输入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>460</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_2
{
border:none;
color:#ffffff;
  background-color: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 1,
        stop: 0 #767aff,
        stop: 1 #83bcff 
    );
border-radius:25;
}


#pushButton_2:hover
{
  background-color: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 1,
        stop: 0 #ec7f9e,
        stop: 1 #9ca2c4 
    );
}</string>
    </property>
    <property name="text">
     <string>下一步</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_3
{
border:1px solid  #e5e7e9;
border-right:none;
border-top-left-radius:25;
border-bottom-left-radius:25;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/icons/acount.png</normaloff>:/icons/acount.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_4
{
border:1px solid  #e5e7e9;
border-right:none;
border-top-left-radius:25;
border-bottom-left-radius:25;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/icons/password.png</normaloff>:/icons/password.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_5
{
border:1px solid  #e5e7e9;
border-right:none;
border-top-left-radius:25;
border-bottom-left-radius:25;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/icons/rightPassword.png</normaloff>:/icons/rightPassword.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
